### ISSUE TITLE
VirtualizedMetroTreeListView Could not be resolved

### ISSUE THREAD

**Issue by MindFlare56** [Anapher/MetroTreeListView#2]
So I tried to do the same thing as the demo in my project.
The problem is that it **can not find resource VirtualizedMetroTreeListView.**

So I ended up downloading the master and trying the same thing. And the demo work on my computer. 
I also noticed **MahApps.Metro is not automatically installed** as a dependence when I download TreeListView from NuGet on my project as it is on the demo version.
**We need to manually install it when installing TreeListView**
After adding the missing NuGet package it still warns me in my project that the 
VirtualizedMetroTreeListView ressource cannot be found.
I ended up finding the problem. I also need to **add the resource dictionary styles** manually inside **App.xaml:**
``` xaml 
<ResourceDictionary.MergedDictionaries>
    <ResourceDictionary Source="pack://application:,,,/MahApps.Metro;component/Styles/Controls.xaml" />
    <ResourceDictionary Source="pack://application:,,,/MahApps.Metro;component/Styles/Fonts.xaml" />
    <ResourceDictionary Source="pack://application:,,,/MahApps.Metro;component/Styles/Colors.xaml" />
    <ResourceDictionary Source="pack://application:,,,/MahApps.Metro;component/Styles/Controls.AnimatedTabControl.xaml" />
    <ResourceDictionary Source="pack://application:,,,/MahApps.Metro;component/Styles/Accents/Blue.xaml" />
    <ResourceDictionary Source="pack://application:,,,/MahApps.Metro;component/Styles/Accents/BaseLight.xaml" />
    <ResourceDictionary Source="pack://application:,,,/TreeListView;component/Resources/TreeListView.xaml" />
</ResourceDictionary.MergedDictionaries>
```

### To sum up
I hope you could add a usage example in the readme file and make it automatically add the required dependencies.
I've found how to make it work so other users could refer to this.
And also... **Good job** I really enjoy a lot your work!
